### PR TITLE
View chat

### PR DESCRIPTION
**Pull request by quanglx-noovado**
## Related Tickets
- [#40176](https://edu-redmine.sun-asterisk.vn/issues/40176)

## WHAT (optional)
221427379_252500776705859_1875189333704456679_n

### CODE REVIEW

**Review by hoanghuee — APPROVED** [review]

**Comment by quanglx-noovado** [comment]
ready

File: resources/js/chat.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                'X-CSRF-TOKEN': $('input[name="_token"]').val()
+            },
+            type: "POST",
+            url: url,
+            data: data,
+            cache: false,
+            success: function (data) {
+                $('#messages').html(data)
+                scrollToBottom()
+                $('#message').val("");
```
Comment (ghost) [inline resources/js/chat.js]:
```suggestion
                $('#messages').html(data);
                scrollToBottom();
                $('#message').val("");
```

File: resources/js/chat.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            $('#chat-messages').append(html);
+            scrollToBottom()
+        } else {
+            let url = $('#list-conversation').data("url");
+            $.ajax({
+                type: "GET",
+                url: url,
+                cache: false,
+                success: function (data) {
+                    $('#list-conversation').html(data)
```
Comment (ghost) [inline resources/js/chat.js]:
```suggestion
                    $('#list-conversation').html(data);
```
fix các chỗ khác nữa nhé

**Comment by quanglx-noovado** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
2 commits

**Comment by quanglx-noovado** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**